rabbitmq/init.sls

```diff
@@ -2,10 +2,10 @@ rabbitmq-server:
   pkgrepo.managed:
     - name: deb http://www.rabbitmq.com/debian/ testing main
     - key_url: http://www.rabbitmq.com/rabbitmq-signing-key-public.asc
+    - require_in:
+      - pkg: rabbitmq-server
   pkg:
-    - installed
-    - require:
-      - pkgrepo: rabbitmq-server
+    - latest
```

Comment:
I've looked through the code some more to determine how `refresh` is used, but I'm still not certain. I think it's probably a no-op unless it's set to `False`, but can't hurt to be explicit. :+1: either way from me.